### PR TITLE
[SOC] Translate the globalParams.md

### PR DESCRIPTION
**Pull request by QuakeWang**
According to this [document](https://dolphinscheduler.apache.org/zh-cn/docs/dev/user_doc/globalParams.html), then translated it.

### CODE REVIEW

File: docs/en-us/dev/user_doc/globalParams.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+Currently, only SQL and SHELL nodes are supported to get parameters.
+
+-    Get the parameter with direction OUT from localParam, and do the following way according to the type of different nodes.
+
+###### SQL node:
+
+The structure returned by the parameter is List<Map<String,String>>.
+
+Where the elements of List are each row of data, the key of Map is the column name, and the value is the value corresponding to the column.
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/globalParams.md:129]: . Where => , where

File: docs/en-us/dev/user_doc/globalParams.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+
+Get the parameter of localParam which is OUT, and exit if it is empty.
+
+Format String as per appeal format (List<Map<String,String>> for SQL, Map<String,String>> for shell).
+
+Assign the parameters with matching values to varPool (List, which contains the original IN's parameters)
+
+-    Format the varPool as json and pass it to master.
+
+-    The master receives the varPool and writes back the parameters that are OUT to the localParam.
```
Comment (ptyp) [inline docs/en-us/dev/user_doc/globalParams.md:161]: The parameters that are OUT would be written into the localParam after the master has received the varPool.

**Review by ptyp — APPROVED** [review]

**Comment by CalvinKirs** [comment]
hi,please resolve conflicts files

**Comment by QuakeWang** [comment]
> hi,please resolve conflicts files

hi, I have resolved the conflict, please merge it : )

**Review by CalvinKirs — APPROVED** [review]
LGTM

**Pull request merged**